README.md

```diff
@@ -163,10 +163,14 @@ bundles:
     service: controller
 plugins:
   envoy_ext_authz_grpc:
-    addr: :9191
-    path: envoy/authz/allow
-    dry-run: false
-    enable-reflection: false
+    addr: :9191 # default `:9191`
+    path: envoy/authz/allow # default: `envoy/authz/allow`
+    dry-run: false # default: false
+    enable-reflection: false # default: false
+    grpc-max-recv-msg-size: 40194304 # default: 1024 * 1024 * 4
+    grpc-max-send-msg-size: 2147483647 # default: max Int
+    skip-request-body-parse: false # default: false
+    enable-performance-metrics: false # default: false. Adds `grpc_request_duration_seconds` prometheus histogram metric 
```

Comment:
@antgubarev here we are stating that by enabling this we'll get a metric called `grpc_request_duration_seconds` which is a histogram metric. This looks good. But below we've defined `grpc_request_duration_seconds` as a `prometheus.Summary` which does not seem right. I would have imagined something like a `prometheus.HistogramVec` similar to what [OPA](https://sourcegraph.com/github.com/open-policy-agent/opa@c954a0c1eef6218b4e7f11ec5785ddc45b5a2141/-/blob/internal/prometheus/prometheus.go?L45) does. This would be a more useful metric than a summary as users typically want to understand the latency distribution of their requests. WDYT?

Reply:
@ashutosh-narkar We've already used `prometheus.HistogramVec` not `Summary` :)

Reply:
https://github.com/open-policy-agent/opa-envoy-plugin/pull/414/files#diff-67213f39b84a013e7db04664cd3ad943b7a237d97b9c6e862d10806ac27def28R244 here we seem to be using `Summary`.